closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -665,6 +698,13 @@ JNIEXPORT jlong JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_loadCrypto
         (NULL == OSSL_EC_KEY_set_public_key) ||
         (NULL == OSSL_EC_KEY_check_key) ||
         (NULL == OSSL_PKCS12_key_gen) ||
+        (NULL == OSSL_ECDSA_do_sign) ||
+        (NULL == OSSL_ECDSA_do_verify) ||
+        (NULL == OSSL_ECDSA_SIG_new) ||
+        (NULL == OSSL_ECDSA_SIG_free) ||
+        (NULL == OSSL_ECDSA_SIG_get0_r) ||
+        (NULL == OSSL_ECDSA_SIG_get0_s) ||
+        (NULL == OSSL_ECDSA_SIG_set0) ||
```

Comment:
I don't think we want these included here (lines 722-729 check them when appropriate).

Reply:
Sorry, I forgot to remove those.